### ISSUE TITLE
Nested enums are not correctly handled

### ISSUE THREAD

**Issue by mishas** [nelfin/pylint-protobuf#16]
Consider the following `blah.proto` file:
```
enum EnumOne {
    UNDEFINED = 0;
    ONE = 1;
    TWO = 2;
}

message Message {
    enum EnumTwo {
        UNDEFINED = 0;
        UNO = 1;
        DOS = 2;
    }
    [...]
}
```
And consider the following code:
```
import blah_pb2

blah_pb2.ONE  # This one is just fine, as expected.
blah_pb2.Message.UNO  # This one is the correct way (will run) but triggers a pylint warning.
blah_pb2.UNO  # This one is incorrect (won't run), but it doesn't trigger a pylint warning.
```

**Comment by nelfin** [comment]
Thanks for the report. I'll see what the interaction is with nested message definitions (which are handled) and nested enum definitions, which seemingly aren't.

**Comment by nelfin** [comment]
`blah_pb2.UNO` didn't warn because it turned out modules were skipped from consideration when checking attributes. I've fixed that now.

**Comment by eugene-kulak** [comment]
@nelfin any progress on this? this is a very common scenario for me

**Comment by nelfin** [comment]
@eugene-kulak: progress on pylint-protobuf? Some. Material progress on this particular issue? No, sorry. Truthfully I don't use protobufs day-to-day so I struggle to make progress on this plugin. I've just started working on it again this last week after a months-long hiatus but I can't guarantee any features or fixes by any given deadline at the moment.

**Comment by nelfin** [comment]
@mishas, @eugene-kulak: I believe all these cases are addressed in 0.14. See https://github.com/nelfin/pylint-protobuf/blob/6593ec1f39bd607d68e30f43b25804ca1e8d71c4/tests/test_enum.py#L119-L173

Are there any other cases you encounter around the behaviour of enum definitions that are unexpected?

**Comment by nelfin** [comment]
I've confirmed on my end that this is fixed, I'll close this issue in a couple of days if there's no update.

**Comment by mishas** [comment]
Hello @nelfin , Thanks for fixing!
I didn't have the chance to test internally, but the unittests look promising.
I think you can close this issue, and worst-case - I'll reopen it :)

Thanks again!

**Comment by nelfin** [comment]
Cheers
